CoreScriptsRoot/CoreScripts/PurchasePromptScript2.lua

```diff
@@ -175,21 +175,32 @@ local ASSET_TO_STRING = {
 local BC_ROBUX_PRODUCTS = { 90, 180, 270, 360, 450, 1000, 2750 }
 local NON_BC_ROBUX_PRODUCTS = { 80, 160, 240, 320, 400, 800, 2000 }
 
-local DIALOG_SIZE = UDim2.new(0, 324, 0, 180)
-local DIALOG_SIZE_TENFOOT = UDim2.new(0, 324*scaleFactor, 0, 180*scaleFactor)
+local DIALOG_SIZE = UDim2.new(0, 324, 0, 240)
```

Comment:
do we know how this looks on a tiny phone? Like an iPhone 4S? maybe try it in the simulator, resolution on that device is 480x320